soc/nordic/common/poweroff.c

```diff
@@ -16,12 +16,18 @@
 #endif
 #if defined(CONFIG_SOC_SERIES_NRF54LX)
 #include <helpers/nrfx_reset_reason.h>
+#include <hal/nrf_memconf.h>
 #endif
 
 #if defined(CONFIG_HAS_NORDIC_RAM_CTRL)
 #include <helpers/nrfx_ram_ctrl.h>
 #endif
 
+#if defined(CONFIG_SOC_SERIES_NRF54LX)
+#define VPR_POWER_IDX 1
```

Comment:
I have confirmed this does not collide with configuration applied by `retained_mem` driver, as general-purpose RAM control is configured via `POWER_IDX==0`, bits 0 to 7